src/main/resources/data/TestData.sql

```diff
@@ -820,9 +820,15 @@ insert into course_decks (course_id, deck_id) values (4, 9);
 insert into course_decks (course_id, deck_id) values (4, 10);
 insert into course_decks (course_id, deck_id) values (5, 11);
 
-INSERT INTO remembering_level (id, number_of_postponed_days) VALUES (1, 1);
-INSERT INTO remembering_level (id, number_of_postponed_days) VALUES (2, 3);
-INSERT INTO remembering_level (id, number_of_postponed_days) VALUES (3, 7);
-INSERT INTO remembering_level (id, number_of_postponed_days) VALUES (4, 14);
-INSERT INTO remembering_level (id, number_of_postponed_days) VALUES (5, 30);
-INSERT INTO remembering_level (id, number_of_postponed_days) VALUES (6, 60);
+INSERT INTO remembering_level (id, order_number, account_id, name, number_of_postponed_days)
```

Comment:
You are inserting the same data both by script and by code?

Reply:
It's needed at least to fill remembering level for admin because it's not created through the program but through the testData.